mappings/net/minecraft/network/packet/c2s/common/CookieReqestResponseS2CPacket.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_9091 net/minecraft/network/packet/c2s/common/CookieReqestResponseS2CPacket
```

Comment:
Typo in the name, be sure to fix both the file name and the content itself

Reply:
Should be fixed now.